docs/source/changelog.rst

```diff
@@ -1,6 +1,18 @@
 =========
 Changelog
 =========
+**Future**
+    * Changes
+        * Add `nltk-data.tar.gz` which contains the nltk corpuses we currently reference :pr:`31`
+        * Add that data to the `nltk` data path at import-time :pr:`31`
+        * Augment `sdist` to first unpack the data tarball, then include data files in the sdist via `MANIFEST.in` :pr:`31`
+        * Add `make clean` command :pr:`31`
```

Comment:
some of these changes are more developer-oriented than user-oriented, maybe we should add a "Development changes" section or something similar

Reply:
@rwedge that's a good point. I'll try to clean this up a bit, and if it still feels weird, let's discuss again.